app/models/project_media.rb

```diff
@@ -430,6 +430,31 @@ def apply_rules_and_actions_on_update
     self.team.apply_rules_and_actions(self, rule_ids)
   end
 
+  def self.handle_fact_check_for_existing_claim(existing_pm,new_pm)
+    if existing_pm.fact_check.blank?
+      existing_pm.appended_fact_check_from(new_pm)
+    elsif existing_pm.fact_check.present?
+      if existing_pm.fact_check.language != new_pm.set_fact_check['language']
+        new_pm.replaced_with_blank_media
```

Comment:
Suggestion: Since this method takes an action, I suggest to name it with the verb name in imperative form, so, `replace_with_blank_media`.